1-development/list_posts_by_rating/app/services/post/search_query_service.rb

```diff
@@ -0,0 +1,15 @@
+class Post
+  class SearchQueryService
+    # Rating Score implementation with lower bound of the Wilson score confidence interval for a Bernoulli parameter
```

Comment:
I used this implementation that I had used previously. This way we can get rid of problems like negative scores _(if we only use `upvotes` - `downvotes`)_ and division problems (if we use `upvotes` / `downvotes` and we don't have a `downvote`)

Reply:
Can you explain this values?

Reply:
For sure. This implementation provides a way to sort a product based on positive and negative ratings. Is a `95% `chance that the real fraction of positive ratings (level of confidence).
![image](https://user-images.githubusercontent.com/934203/106899660-647dd480-66d4-11eb-8dd8-b7e4f1e88654.png)

`p̂ `​= Number of positive ratings / Total Ratings
`n` = Total ratings
`Z*Z(α/2)` =  Quantil of the standard normal distribution